### ISSUE TITLE
README.md revamp

### ISSUE THREAD

**Issue by crodriguezvega** [cosmos/ibc-go#1747]
Possible content for an improved README.md. Each of this items is a section title with a brief explanation of the content under it.

- [ ] Introduction
  - Project description (I guess we can use the text that we already have, but maybe we could add as well a link to Adi's ELI5 blog post - as we have done [in the spec repo](https://github.com/cosmos/ibc#what-is-ibc))
- [ ] Vision
  -  Explain the long-term goals and focus of the repo.
- [x] Roadmap
- [x] Releases
  - Include link to version matrix.
- [x] Ecosystem
  - The apps/middlewares/light-clients registry.
- [x] Contributing
  - Summary of contributing guidelines and link to full text.
  - Maybe a collage of small pictures of all the contributors to the project? (for example [like this](https://github.com/vuejs/vue#contribution))
- [x] Support
  - How to reach us and explain what each medium is most suited for (Discord, Telegram, GitHub discussions, etc).
- [x] License
- [ ] Security
  - Link to bug bounty program and explain how to submit an issue.
- [x] Resources
  - List of links to the spec, docs, relevant wiki pages, other relevant project (hermes, rly), etc. 
- [ ] A paragraph at the end explaining that the team at Interchain GmbH is the primary maintainer and that funding comes from the ICF 

Please comment with any other content that we should add.

**Comment by charleenfei** [comment]
i think there should be a more direct link between the high level categories on the roadmap and the project boards in the repo, maybe an additional project board which is just an "epic tracker" that serves as that link, or direct links on the roadmap.

**Comment by crodriguezvega** [comment]
> i think there should be a more direct link between the high level categories on the roadmap and the project boards in the repo, maybe an additional project board which is just an "epic tracker" that serves as that link, or direct links on the roadmap.

The roadmap file contains already a bunch of links to epic issues, milestones or boards (if available at the moment of writing the roadmap). Would this be enough or you have something else in mind?

The epic tracker board is something that we can also add.

**Comment by charleenfei** [comment]
I guess I mean a one stop shop view from the project board to the roadmap (ie: epic tracker board). The roadmap file contains links but they are in the middle of text, i guess i meant something more like you can track the progress of whats happening on the roadmap.

**Comment by colin-axner** [comment]
yay! Super excited for this :)
